nodejs/eks/nodegroup.ts

```diff
@@ -315,32 +315,41 @@ ${customUserData}
 `;
         });
 
-    let amiId: any = args.amiId!;
+    let amiId: pulumi.Input<string> = args.amiId!;
     const version: pulumi.Input<string> = args.version!;
     if (args.amiId === undefined) {
         const filters: { name: string; values: string[]}[] = [
+            // Filter to target Linux arch
             {
                 name: "description",
                 values: [ "*linux*", "*Linux*" ],
             },
+            // Filter to target EKS Nodes
             {
-                name: "description",
-                values: [ "*k8s*/bin/linux/amd64*"],
+                name: "name",
+                values: [ "amazon-eks-node-*"],
+            },
+            // Filter to target General, non-GPU image class
+            {
+                name: "manifest-location",
+                values: [ "*amazon-eks-node*"],
             },
         ];
 
         if (version !== undefined) {
             filters.push(
+                // Filter to target a specific k8s version
                 {
-                    name: "name",
-                    values: [ "amazon-eks-node-" + version + "*" ],
+                    name: "description",
+                    values: [ "*k8s*" + version + "*" ],
```

Comment:
Does this approach of having two filters on `description` work?  I assume so since you’ve tried this - but I hadn’t expected it would.

Reply:
Yup, you can apparently chain them: https://gist.github.com/metral/4ba5cfe72fffe3c30dc0a57cbd08bd97